symfony/fabfile.py

```diff
@@ -28,7 +28,7 @@
 
 
 @task
-def main(repo, repourl, branch, build, buildtype, siteroot, keepbuilds=10, url=None, buildtype_override=False, ckfinder=False, keepbackup=False, migrations=False, cluster=False, with_no_dev=True, php_ini_file=None):
+def main(repo, repourl, branch, build, buildtype, siteroot, keepbuilds=10, url=None, buildtype_override=False, ckfinder=False, keepbackup=False, migrations=False, cluster=False, with_no_dev=True, php_ini_file=None, parameters_yml=True, env_file=False):
```

Comment:
Python/Fabric is iffy because everything passed in from the fab command is treated as a string. So my concern here is if someone doesn't want to use a parameters.yml file and set parameters_yml=False in the Jenkins job, that'll be treated as "False". Then, the `if parameters_yml:` check will return True.

We may need to do some logic beforehand, at the start of the function:

```python
use_parameters_yml = True
if parameters_yml == 'False':
  use_parameters_yml = False

use_env_file = False
if env_file == 'True':
  use_env_file = True
```

or something like that.

Reply:
Updated